modin/pandas/test/test_dataframe.py

```diff
@@ -1385,14 +1385,38 @@ def test_drop_api_equivalence():
         modin_df.drop(axis=1)
 
 
-@pytest.mark.skip(reason="Defaulting to Pandas")
-@pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
-def test_drop_duplicates(data):
-    modin_df = pd.DataFrame(data)
-    pandas_df = pandas.DataFrame(data)  # noqa F841
+def test_drop_duplicates():
```

Comment:
It might be good to do:

```python
@pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
def test_drop_duplicates(data):
```

It will require some small changes in the test code, but it will test on a wider variety of edge case DataFrames.

Reply:
That sounds like a good idea, though I think we need the full `test_data` dictionary since including the "float_data" column in searching for duplicates effectively guarantees that there will never a duplicate